contracts/FlowIDTableStaking.cdc

```diff
@@ -924,7 +929,9 @@ pub contract FlowIDTableStaking {
         for nodeID in FlowIDTableStaking.getNodeIDs() {
             let nodeRecord = FlowIDTableStaking.borrowNodeRecord(nodeID)
 
-            if nodeRecord.tokensStaked.balance >= self.minimumStakeRequired[nodeRecord.role]!  {
+            // Access nodes do not have a minimum, so this makes sure they are included when they are staked,
+            // but not if they are equal to their minimum, which is zero
+            if nodeRecord.tokensStaked.balance > 0.0 {
```

Comment:
but why it no longer needs to check against `minimumStakeRequired`?

Reply:
Because before, it would consider access nodes a staked node even if their tokens staked were zero, because the minimum for access nodes is zero. Now, we can just say `> 0.0` because we know that if a node has any tokens staked, it will definitely be above the minimum, because the `endStaking` function enforces that.

Reply:
It's worth making a comment to that effect, as I had the same question.

Given that the idiom elsewhere is to check `nodeRecord.role == UInt8(5)` it might be clearer to do so here as well.

Reply:
So the comment that I already put there isn't sufficient? That is why I included it

Reply:
Sorry, my brain stopped working briefly. Yes that's great.

Reply:
I make the check much more clear about what it is checking, and I also clarified it in the comment. Can you let me know if it looks good @robmyers @zhangchiqing ?

Reply:
I wonder if we need to keep the check here consistent 
https://github.com/onflow/flow-core-contracts/pull/62/files#diff-65336be374bb3fc9ad7b822243e065a389d73e758c7c16223e52fc3181cea59bR914

If so, it worth to extract into a function